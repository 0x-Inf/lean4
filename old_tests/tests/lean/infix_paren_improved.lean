constant f : Π {α : Type}, α → α → α

infix `**`:60 := f

#check (**)
