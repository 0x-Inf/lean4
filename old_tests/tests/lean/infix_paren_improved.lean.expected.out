f : ?M_1 → ?M_1 → ?M_1
